response_operations_ui/views/messages.py

```diff
@@ -480,6 +464,7 @@ def view_selected_survey(selected_survey):  # noqa: C901
     limit = request.args.get("limit", default=10, type=int)
     conversation_tab = request.args.get("conversation_tab", default="open")
     ru_ref_filter = request.args.get("ru_ref_filter", default="")
+    thread_id = request.args.get("thread_id", default="")
```

Comment:
doesn't need the default, get has a standard default of None, which will be treated the same as empty string